services/npm/npm-unpacked-size.service.js

```diff
@@ -0,0 +1,65 @@
+import Joi from 'joi'
+import prettyBytes from 'pretty-bytes'
+import { BaseJsonService, pathParam } from '../index.js'
+import { packageNameDescription } from './npm-base.js'
+
+export default class NpmUnpackedSize extends BaseJsonService {
```

Comment:
The reason I've specifically used `BaseJsonService` instead of `NpmBase` is because I, for some reason, could not quite much understand it. It has its own NPM Registry parsing logic, some other services of NPM has different URLs, etc. It's been 2 days since I've opened the issue myself just to try to wrap my head around the existing NPM services implementing `NpmBase`. At the end of the day, I've found it much more easier and clean to implement `BaseJsonService` directly.

If, however, it is strictly required, I will try to implement it in `NpmBase` way.

Reply:
actually, nvm, changing to `NpmBase` worked flawlessly